analyzers/src/SonarAnalyzer.VisualBasic/Metrics/VisualBasicExecutableLinesMetric.cs

```diff
@@ -97,6 +97,7 @@ private bool FindExecutableLines(SyntaxNode node)
                     case SyntaxKind.WhileStatement:
 
                     case SyntaxKind.IfStatement:
+                    case SyntaxKind.ElseIfStatement:
```

Comment:
See this commit and the modified UT for more context: [`882a1e7` (#5082)](https://github.com/SonarSource/sonar-dotnet/pull/5082/commits/882a1e757a5a2eeba160d7696278cc6f4e489963)

Reply:
This should be in the release notes as it's a functional change

Reply:
See #5109